src/app/shared/utils/consts.ts

```diff
@@ -171,6 +174,16 @@ export const Consts = {
                 value: 'ibm'
             }
         ],
+        resources:{
+            volumes : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            pools : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            controllers : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            ports : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc'],
+            disks : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            qtrees : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            filesystems : ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc', 'fake_driver'],
+            shares: ['vmax', 'unity', 'vnx', 'oceanstor', '3par', 'vsp', 'storwize_svc']
```

Comment:
Quotas also we need to add.. for displaying qouata we can either put it as different tab, or is it possible to associate with  filestem and qtree objects?

Reply:
I do not have the details for adding Quotas. We can probably take it up as a subsequent task.

Reply:
@NajmudheenCT This has been logged as an issue #539